examples/ssd1306_128x64_i2c/ssd1306_128x64_i2c.ino

```diff
@@ -56,7 +56,7 @@ void setup() {
   Serial.begin(9600);
 
   // SSD1306_SWITCHCAPVCC = generate display voltage from 3.3V internally
-  if(!display.begin(SSD1306_SWITCHCAPVCC, 0x3D)) { // Address 0x3D for 128x64
+  if(!display.begin(SSD1306_SWITCHCAPVCC, 0x3C)) { // Address 0x3D for 128x64
```

Comment:
This leaves a stale value in the comment. It might be worthwhile to make a real constant out of it up near the kind of related `OLED_RESET` definition, and let the name of the constant be self-documenting instead.

    #define OLED_I2C_ADDR 0x3C
    ....
    if(!display.begin(SSD1306_SWITCHCAPVCC, OLED_I2C_ADDR)) {


If this change is made, maybe we'd want the other example sketches to have the same form?

Reply:
Good idea.But I think just change the address and comment is enough.